api/v1alpha1/api.go

```diff
@@ -227,6 +227,25 @@ type AIGatewayRouteRule struct {
 	//
 	// +optional
 	Timeouts *gwapiv1.HTTPRouteTimeouts `json:"timeouts,omitempty"`
+
+	// OwnedBy represents the owner of the running model serving by the backend,
+	// which will be exported as the field of "OwnedBy" in openai-compatible API "/models".
+	//
+	// Default to "Envoy AI Gateway" if not set.
+	//
+	// +optional
+	// +kubebuilder:default="Envoy AI Gateway"
+	OwnedBy *string `json:"ownedBy,omitempty"`
```

Comment:
I would like to emphasize that this is only for "/models" endpoint. so I would suggest to prefix "models"
```suggestion
	ModelsOwnedBy *string `json:"modelsOwnedBy,omitempty"`
```